services/management/grpc/actions_server.go

```diff
@@ -253,6 +253,42 @@ func (s *actionsServer) StartMySQLShowIndexAction(ctx context.Context, req *mana
 	}, nil
 }
 
+// StartMySQLShowIndexAction starts PostgreSQL SHOW CREATE TABLE Action.
```

Comment:
Not MySQL, and not ShowIndex.

It looks like several linter warnings on the same line confuse @golangcibot. Please run it locally:
```
golangci-lint run --new-from-rev=PMM-2.0
```